recipes/deepspeed/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "deepspeed" %}
+{% set version = "0.6.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/deepspeed-{{ version }}.tar.gz
+  sha256: ad2de46d6609798fa3525ef8eb0e1ddfb66e06983b0af967b462b1511f21cc9c
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - pytorch
+  run:
+    # - hjson
```

Comment:
TODO need to add https://github.com/hjson/hjson to conda-forge to fix `ModuleNotFoundError: No module named 'hjson'` at https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=509480&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=2575